llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -610,6 +610,14 @@ void BranchInst::dump() const {
 }
 #endif // NDEBUG
 
+void LoadInst::setVolatile(bool V) {
+  auto &Tracker = Ctx.getTracker();
+  if (Tracker.isTracking())
+    Tracker.track(
+        std::make_unique<SetVolatile>(cast<Instruction>(this), Tracker));
```

Comment:
Hmm, this `cast<Instruction>(this)` shouldn't be needed as `this` is already an Instruction. So replacing it with `this` should work too.